elements/rh-code-block/rh-code-block.css

```diff
@@ -279,3 +286,76 @@
   margin-block: var(--rh-space-lg, 16px);
   gap: var(--rh-space-md, 8px);
 }
+
+.on,
+.on.light {
+  --_default-color: var(--rh-color-gray-95, #151515);
+  --_selected-text-background: var(--rh-color-blue-10, #e0f0ff);
+  --_comment-color: var(--rh-color-gray-60, #4d4d4d);
+  --_comment-block-color: var(--rh-color-gray-60, #4d4d4d);
+  --_doctype-color: var(--rh-color-gray-60, #4d4d4d);
+  --_cdata-color: var(--rh-color-gray-60, #4d4d4d);
+  --_punctuation-color: var(--rh-color-gray-40, #a3a3a3);
+  --_namespace-color: var(--rh-color-gray-95, #151515);
+  --_property-color: var(--rh-color-purple-50, #5e40be);
+  --_tag-color: var(--rh-color-purple-50, #5e40be);
+  --_boolean-color: var(--rh-color-purple-50, #5e40be);
+  --_number-color: var(--rh-color-purple-50, #5e40be);
+  --_constant-color: var(--rh-color-purple-50, #5e40be);
+  --_symbol-color: var(--rh-color-purple-50, #5e40be);
+  --_deleted-color: var(--rh-color-purple-50, #5e40be);
+  --_function-name-color: var(--rh-color-purple-50, #5e40be);
+  --_selector-color: var(--rh-color-teal-60, #147878);
+  --_attr-name-color: var(--rh-color-teal-60, #147878);
+  --_string-color: var(--rh-color-teal-60, #147878);
+  --_character-color: var(--rh-color-teal-60, #147878);
+  --_built-in-color: var(--rh-color-teal-60, #147878);
+  --_inserted-color: var(--rh-color-teal-60, #147878);
+  --_operator-color: var(--rh-color-yellow-60, #96640f);
+  --_entity-color: var(--rh-color-yellow-60, #96640f);
+  --_url-color: var(--rh-color-yellow-60, #96640f);
+  --_at-rule-color: var(--rh-color-blue-60, #004d99);
+  --_attr-value-color: var(--rh-color-blue-60, #004d99);
+  --_keyword-color: var(--rh-color-blue-60, #004d99);
+  --_function-color: var(--rh-color-red-60, #a60000);
+  --_class-name-color: var(--rh-color-red-60, #a60000);
+  --_regex-color: var(--rh-color-orange-60, #9e4a06);
+  --_important-color: var(--rh-color-orange-60, #9e4a06);
+  --_variable-color: var(--rh-color-orange-60, #9e4a06);
+}
+
+.on.dark {
+  --_default-color: var(--rh-color-gray-20, #e0e0e0);
+  --_selected-text-background: var(--rh-color-gray-95, #151515);
+  --_comment-color: var(--rh-color-gray-50, #707070);
+  --_comment-block-color: var(--rh-color-gray-50, #707070);
+  --_doctype-color: var(--rh-color-gray-50, #707070);
+  --_cdata-color: var(--rh-color-gray-50, #707070);
```

Comment:
This color does not pass WCAG color contrast guidelines on dark.

Reply:
CC @marionnegp

Reply:
`--rh-color-gray-60, #4D4D4D` for all of those